terraform/helm/aptos-node/values.yaml

```diff
@@ -5,6 +5,8 @@ chain:
   era: 1
   # -- Chain ID
   chain_id: 4
+  # -- Epoch duration
+  epoch_duration_secs: 600
```

Comment:
`aptos-node` doesn't have the value `chain.epoch_duration_secs`. I think you mean to put it in `testsuite/forge/src/backend/k8s/helm-values/genesis-values.yaml` such that the override is only in forge.